packages/documentation/copy/en/declaration-files/By Example.md

```diff
@@ -119,7 +119,7 @@ _Declaration_
 You can use a type alias to make a shorthand for a type:
 
 ```ts
-type GreetingLike = string | (() => string) | MyGreeter;
+type GreetingLike = string | (() => string) | Greeter;
```

Comment:
I'm not sure this much matters for the example; theoretically both are present in this sample and are functionally equivalent.